app/src/main/res/layout/item_positive_diagnosis.xml

```diff
@@ -14,7 +14,8 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintVertical_bias="0.05"
-        app:srcCompat="@drawable/ic_info_red" />
+        app:srcCompat="@drawable/ic_info_red"
+        android:contentDescription="@string/warn_icon_content_description"/>
```

Comment:
In this context, this icon means that the diagnosis is not verified, not just a waning.
If we really want to add content description to this image we need to do that here:
https://github.com/covid19risk/covidwatch-android-en/blob/f0a0de716abd117271dade2834873977a38f3e2d/app/src/main/java/org/covidwatch/android/ui/reporting/PositiveDiagnosisViewHolder.kt#L25-L35

But, this view already has a text description near it so we could avoid setting description on this `ImageView`

Reply:
If we have relevant content in the text view, that's good enough for me. I'm working in resource files, so it looks out of context there, but we don't need the descriptor if their is runtime context.